.github/workflows/build.yml

```diff
@@ -142,10 +142,13 @@ jobs:
         echo "Placed the image in ${target_dir}/${target_name}.bz2 with bmap file"
 
     - name: Build Image (MrBeam Variant)
+      env:
+        PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
Is it necessary do add it here? I thought you could just use it directly, same as with `secrets.SSH_BEAMOS01_RSA`

Reply:
It didnt work so I tried a lot 😅😅